user-service/src/main/java/org/arpha/security/oauth2/CustomOAuth2SuccessHandler.java

```diff
@@ -38,8 +39,20 @@ public void onAuthenticationSuccess(HttpServletRequest request, HttpServletRespo
         } else {
             userResponse = userService.findUserByEmail(email);
         }
-        String jwtToken = jwtUtils.generateToken(email);
-        LoginResponse loginResponse =  authMapper.toLoginResponse(userResponse, jwtToken);
+
+        LoginResponse loginResponse = getLoginResponse(email, userResponse);
         response.getWriter().write(objectMapper.writeValueAsString(loginResponse));
     }
+
+    private LoginResponse getLoginResponse(String email, UserResponse userResponse) {
```

Comment:
Didn't we agree to use mappers?

Reply:
We agreed not to find sensible logic in this mappers, for me creating a LoginResponse in this specific case is a sensible logic that better be exposed